packages/react-look/modules/plugins/friendlyClassName.js

```diff
@@ -17,7 +17,13 @@ export default function friendlyClassNames({ StyleContainer, Component, element,
       if (classMapping.has(cls)) {
         className += className + className !== '' ? ' ' : '' + classMapping.get(cls)
       } else {
-        const newClass = classNameTemplate(cls, Component, element)
+        let newClass
+        // checking if config is undefined to avoid breaking existing test
```

Comment:
@moaazsidat  Maybe we should just adjust the old test and add `config: {}` to avoid crashing, because Look ensures a config always is provided anyway (even if it only is `{}`).